lib/Phpactor.php

```diff
@@ -76,9 +76,12 @@ public static function boot(InputInterface $input, string $vendorDir): Container
         $config = $loader->load();
         $config[CoreExtension::PARAM_COMMAND] = $input->getFirstArgument();
         $config[FilePathResolverExtension::PARAM_APPLICATION_ROOT] = self::resolveApplicationRoot();
-        $config[IndexerExtension::PARAM_STUB_PATHS] = [
+        $config[IndexerExtension::PARAM_STUB_PATHS] = isset($config[IndexerExtension::PARAM_STUB_PATHS])
+            ? array_push(
+                $config[IndexerExtension::PARAM_STUB_PATHS],
                 self::resolveApplicationRoot() . '/vendor/jetbrains/phpstorm-stubs'
-        ];
+            )
+            : [ self::resolveApplicationRoot() . '/vendor/jetbrains/phpstorm-stubs' ];
```

Comment:
```
$config[IndexerExtension::PARAM_STUB_PATHS] = array_merge([
     IndexerExtension::PARAM_STUB_PATHS = [],
], $config);
$config[IndexerExtension::PARAM_STUB_PATHS][] = self::resolveApplicationRoot() . '/vendor/jetbrains/phpstorm-stubs';
```

maybe better? finally we should do something else, but this is OK for now.

Reply:
Yes, better :)